src/components/shared/cellRenderers/linkRenderer/linkRenderer.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from "react";
+import { NavLink } from 'react-router-dom';
+
+import { Svg } from 'components/shared/svg/svg';
+import { svgs } from 'utilities';
+
+import '../cellRenderer.css';
+
+export const LinkRenderer = ({ uri, context: { t }, svgPath }) => {
+  return (
+    <div className="pcs-renderer-cell">
+      <NavLink to={uri} className="pcs-renderer-link">
```

Comment:
Rename `uri` to `to` to maintain a consistent `to` `path` scheme following the `react-router` naming pattern.